ddp-workspace/projects/ddp-pancan/src/app/components/page-with-sections/page-with-sections.component.html

```diff
@@ -1,5 +1,5 @@
 <main class="page" [ngClass]="isAboutUsPage ? 'about-us': 'scientific-research'">
-    <section *ngFor="let section of (source + '.Sections') | translate; let i = index;" [class]="'section section-' + i">
+    <section *ngFor="let section of (source + '.Sections') | translate; let i = index;" [class]="'section section-' + i "[id] = "'section-' + i">
```

Comment:
Let's moved the `section-i` to the title block so to avoid the padding/spacing above the element and draw attention a bit better to the list of projects.

```suggestion
    <section *ngFor="let section of (source + '.Sections') | translate; let i = index;" [class]="'section'">
```